src/postgraphql/graphiql/graphiql.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import { render } from 'react-dom'
+import GraphiQL from 'graphiql'
+import { buildClientSchema, introspectionQuery } from 'graphql'
+import 'graphiql/graphiql.css'
```

Comment:
Could we move the CSS to just be a `<link>` tag in the HTML? This removes the need for `style-loader` and `css-loader`.